tests/image-stream/tester.js

```diff
@@ -43,11 +43,14 @@ const deleteIfExists = (file) => {
   });
 };
 
-exports.expectError = function(file, errorMessage) {
+exports.expectError = function(file, errorMessage, errorDetail) {
```

Comment:
Should it have a default value, for tests that don't use it? Or do unused parameters in JS automatically default to `null` or something?